src/sbt-test/debian/override-loader-functions/build.sbt

```diff
@@ -0,0 +1,26 @@
+enablePlugins(JavaServerAppPackaging, JDebPackaging, SystemdPlugin)
+
+// TODO change this after #437 is fixed
+daemonUser in Linux := "root"
+daemonGroup in Linux := "app-group"
+
+mainClass in Compile := Some("empty")
+
+name := "debian-test"
+version := "0.1.0"
+maintainer := "Mitch Seymour <mitchseymour@gmail.com>"
+
+packageSummary := "Test debian package"
+packageDescription := """A fun package description of our software,
+  with multiple lines."""
+
+TaskKey[Unit]("check-loader-functions") <<= (target, streams) map { (target, out) =>
+  val extracted = target / "tmp" / "extracted-package"
+  extracted.mkdirs()
+  Seq("dpkg-deb", "-e", (target / "debian-test_0.1.0_all.deb").absolutePath, extracted.absolutePath).!
+
+  val script = IO.read(extracted / "postinst")
+  
+  assert(script.contains("# right systemd template"), s"override script wasn't picked, script is\n$script")
```

Comment:
This checks that the `loader-functions` has be correctly inserted. I think this check is enough, even though I wouldn't my checking that the `wrong systemd template` isn't contained in the script.